Source/Particles/Collision/BinaryCollision/ElasticCollisionPerez.H

```diff
@@ -15,23 +15,28 @@
 #include <AMReX_Random.H>
 
 
-/** \brief Prepare information for and call
- *        UpdateMomentumPerezElastic().
+/** Prepare information for and call UpdateMomentumPerezElastic().
+ *
```

Comment:
```suggestion
 *
 * @tparam T_index type of index arguments
 * @tparam T_R type of floating point arguments
 * @tparam SoaData_type type of the "struct of array" for the two involved species
```
Do we really need T_R ? Shouldn't we use `amrex::Real` or something like that?

Reply:
I think it should be `amrex::ParticleReal` rather than `amrex::Real`, but I don't have a strong preference between `amrex::ParticleReal` and a template parameter.

Reply:
Sounds good - if we know the type then let's use that instead of a template. That keeps the code readable and allows us (usually) to split into H/cpp.